roles/openshift_excluder/tasks/verify_excluder.yml

```diff
@@ -5,25 +5,30 @@
 # - openshift_upgrade_target
 - block:
   - name: Get available excluder version
-    command: >
-      {{ repoquery_cmd }} --qf '%{version}' "{{ excluder }}"
-    register: excluder_version
-    failed_when: false
-    changed_when: false
+    include_role:
+      name: rpm_version
+    vars:
+      rpm_name: "{{ excluder }}"
```

Comment:
`excluder` appears to be a magic variable because we don't really know where it comes from.  Again I feel there is a better way to do this than call out to another role to set a var and then come back.  Can the rpm_version tasks just be included in this role?